.github/workflows/build_msi.yml

```diff
@@ -5,48 +5,27 @@ on:
   push:
     branches:
     - 'main'
-    - '3.11'
-    - '3.10'
-    - '3.9'
-    - '3.8'
-    - '3.7'
+    - '3.*'
```

Comment:
This might be a bit too generic since it might match branches like `3.x-mybranch`.  However we are trying to limit the number of active branches on the main repo, it shouldn't be a concern.